### DISCUSSION TITLE
Understanding the Renovate schedules

### DISCUSSION THREAD

**Discussion by shenke-adsk** [renovatebot/renovate#26192] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.100.0

### If you're self-hosting Renovate, select which platform you are using.

GitHub Enterprise Server

### What is your question?

We are using Renovate On-Prem image 5.1.0. Currently, we set the Renovate schedule to every 12 hours: `SCHEDULER_CRON = "0 */12 * * *"`. Some people are reporting their repositories never receive Renovate updates. Some of them use cron schedule and some use later schedule in their repositories. I would like to understand better how Renovate is scheduled to run. The documentation on this is rather unclear: https://docs.renovatebot.com/key-concepts/scheduling/ I have many questions on this. 

1. What is the default schedule for Renovate On-Prem? We are running it as a GitHub Enterprise app. 
2. Setting it to `SCHEDULER_CRON = "0 */12 * * *"`, does it mean Renovate gets run every 12 hours globally on all installed repos? 
3. If a repo has its own schedule, for example `"extends": ["config:base", ":prHourlyLimit4", ":automergeMinor", "schedule:weekends"],`, Renovate will run on this repo every Saturday and Sunday 4 times? 
4. What is the behaviour on this repo schedule? `"schedule": ["* 0 * * 2"]`, say Renovate runs on 2am and 14pm every day, then it will never process this repo?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
Hi, Renovate On-Prem has been renamed to Renovate Community Edition and a new major release has been issued. Renovate On-Prem is no longer supported. Renovate CE is supported here: https://github.com/mend/renovate-ce-ee
